### PR TITLE
fix alibi bias

### LINKED ISSUE #29: It seems that `visualize_attention_scores` can only visualize either mask-mod-only or score-mod-only
https://github.com/pytorch-labs/attention-gym/blob/75867424a1d4391bff49527029d3612a09dd67e2/attn_gym/utils.py#L68

### PR DESCRIPTION
**Pull request by kashif**
https://x.com/zmkzmkz/status/1863488773641896025

also fixes #29 

alibi_score_mod

### CODE REVIEW

File: examples/flex_attn.ipynb [hunk 1]
```diff
@@ -700,13 +700,13 @@
...
     "def alibi_and_causal_closure(score, b, h, q_idx, kv_idx):\n",
-    "    bias = alibi_bias[h] * (q_idx - kv_idx)\n",
+    "    bias = alibi_bias[h] * (kv_idx - q_idx)\n",
     "    return score + bias\n",
     "\n",
     "\n",
     "def alibi_and_causal_functional(score, b, h, q_idx, kv_idx):\n",
     "    scale = torch.exp2(-((h + 1) * 8.0 / H))\n",
-    "    bias = (q_idx - kv_idx) * scale\n",
+    "    bias = (kv_idx -q_idx) * scale\n",
```
Comment (kashif) [inline examples/flex_attn.ipynb:709]:
```suggestion
    "    bias = (kv_idx - q_idx) * scale\n",
```
